namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -356,7 +357,17 @@ public void run() {
 					if (g == null) {
 						createGroup(NameLayerPlugin.getSpecialAdminGroup(), null, null);
 					} else {
-						removeAllMembers(g.getName());
+						try (Connection connection = db.getConnection();PreparedStatement removeAllMembers = connection.prepareStatement("delete fm.* from faction_member fm "
+								+ "inner join faction_id fi on fi.group_id = fm.group_id where fi.group_name =?;"))  {
+							removeAllMembers.setString(1, g.getName());
+							removeAllMembers.execute();
+						} catch (SQLException e) {
+							e.printStackTrace();
+						}
+						
+						
+						cleanExecute("delete fm.* from faction_member fm inner join groupPlayerTypes gpt on gpt.type_id = fm.type_id "
```

Comment:
Removing all members used to be a PreparedStatement which had its only use in this upgrade, so I  removed the PreparedStatement and hardcoded the query instead here, might as well have it here if it's only used here. The query is just copied over from the previous PreparedStatement, it will delete all members of a group for a given name, not sure what you mean with it's incomplete. Functionality wise this should be exactly the same as what it was previously.